src/components/VideoCard/Card.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+
+function VideoCard({ src, title, description }) {
+  return (
+    <Card
+      style={{ width: '18rem', height: '30rem', marginBottom: '1rem', marginTop: '1rem' }}
+    >
```

Comment:
You should avoid declaring inline styles. Please try to extend your Bootstrap Card using styled-components and apply your custom styles there like this:

```suggestion
import BootstrapCard from 'react-bootstrap/Card';

const Card = styled(BootstrapCard)`
    width: 18rem;
    // ...other custom css styles
`;

function VideoCard({ src, title, description }) {
  return (
    <Card>
```